domains/zicor.json

```
@@ -0,0 +1,12 @@
+{
+	"description": "chat app",
+	"owner": {
+	  "username": "ZicorXXIX",
+	  "email": "manoj19.sinnur@gmail.com",
+    "discord":".zicor"
+	},
+	"record": {
+	  "CNAME": "https://jolt-sepia.vercel.app/"
```

site seems to be non-developer related?
CNAME can't contain protocol or `/`
```suggestion
	  "CNAME": "jolt-sepia.vercel.app"
```